models/Discord/Commands/DiscordCommandGh0sT.js

```diff
@@ -0,0 +1,23 @@
+const DiscordCommand = require('../DiscordCommand.js');
+
+class DiscordCommandGh0sT extends DiscordCommand {
+
+   constructor(subsystem) {
+    gh0st"gh0st", "Spooky Boi", undefined, subsystem, true);
```

Comment:
# 🤔

Reply:
LEAVE ME ALONE I TYPED THIS UP IN 5 MINS

Reply:
WHY MUST I TRY TO CUSTOM EVERYTHING I DO, LET ME FUCKING GO ALL THE WAY BACK TO MY COMPUTER THEN, I HATE MYSELF